gemfiles/Gemfile.rails-5-0-stable

```diff
@@ -0,0 +1,16 @@
+source 'https://rubygems.org'
+
+gemspec :path => '..'
+
+platforms :rbx do
+  gem 'rubysl', '~> 2.0'
+  gem 'rubinius-developer_tools'
+end
+
+gem 'country_select', '~> 2.5.2'
+gem 'railties', '5.0.0'
```

Comment:
The default `Gemfile` is already targeting the 5.0.0 releases, so we don't need a new Gemfile for it.